src/solvers/smt2_incremental/smt_sorts.h

```diff
@@ -87,6 +87,16 @@ class smt_bit_vector_sortt final : public smt_sortt
   std::size_t bit_width() const;
 };
 
+class smt_array_sortt final : public smt_sortt
+{
+public:
+  explicit smt_array_sortt(
+    const smt_sortt &index_sort,
+    const smt_sortt &element_sort);
+  const smt_sortt &get_index_sort() const;
+  const smt_sortt &get_element_sort() const;
```

Comment:
Could you rename `get_index_sort` and `get_element_sort` to `element_sort` and `element_sort` respectively? This would make it more consistent with the naming conventions used for getters elsewhere such as `smt_bit_vector_sortt::bit_width`. This comment should have been in my initial review, but it went awol somewhere, sorry.